src/dnsimple-test/Services/DomainsEmailForwardsTest.cs

```diff
@@ -107,8 +107,8 @@ public void EmailForwardsData()
         }
 
         [Test]
-        [TestCase(1010, "228963",
-            "https://api.sandbox.dnsimple.com/v2/1010/domains/228963/email_forwards")]
+        [TestCase(1010, "235146",
```

Comment:
I'm just curious: are these changes required because the source fixture has changed or is there any other reason?

Reply:
@ggalmazor yeah. The content of the fixtures has been changed, so we need to update the tests as well